config/development.toml

```diff
@@ -584,6 +584,7 @@ source = "logs"
 
 [events.kafka]
 brokers = ["localhost:9092"]
+fraud_check_analytics_topic= "hyperswitch-fraud-check-events"
```

Comment:
raise hyperswitch-cloud PR for adding these topics to envs
ex: https://github.com/juspay/hyperswitch-cloud/pull/5266/

Reply:
Done